packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md

```diff
@@ -48,8 +51,10 @@ When you enable a self-service registration policy, Okta enforces uniqueness for
 1. If this is your first time on this page, click **Enable Registration**. Otherwise, click **Edit**.
 1. In the **ACCOUNT** section, you can add a sign-up link and assign the user to an existing group automatically:
 
-   * **Add to Sign-In Widget:** Optional. Select this option if you want to add a sign-up link to your Okta-hosted sign-in page. By adding a sign-up link to Okta Sign-In Widget, you eliminate the need to configure a link that uses JavaScript in the Customize Sign-In Page editor.
-   * **Assign to group:** Optional. Enter the existing group name that users are automatically added to when they self-register. Group membership determines which password policy is applied during registration. If no group is specified, Okta applies the Default password policy.
+   * **Add to Sign-In Widget:** Optional. Select this option if you want to add a sign-up link to your Okta-hosted sign-in page. By adding a sign-up link to the Okta Sign-In Widget, you don't need to configure a link that uses JavaScript in the editor.
+   * **Assign to group:** Optional. Enter the group name that users are automatically added to when they self-register. Group membership determines which password policy is applied during registration. If no group is specified, Okta applies the default password policy even if that policy is applied for non-Okta sourced users.
```

Comment:
```suggestion
   * **Assign to group:** Optional. Enter the group name that users are automatically added to when they self-register. Group membership determines which password policy is applied during registration. If no group is specified, Okta applies the default password policy even if that policy is applied for non-Okta-sourced users.
```

Reply:
I mean, the hyphen is correct, but now it looks like a word centipede. I'm good with whatever you think is best.